src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+    private String color;
```

Comment:
The 'color' field in the 'IsoscelesTrapezoid' class is redundant since the color is already managed by the parent 'Figure' class. You should remove this field and use the 'getColor()' method inherited from 'Figure' to access the color.